src/secret/ledger_nano.rs

```diff
@@ -279,6 +313,7 @@ impl LedgerSecretManager {
 
         log::info!("get_ledger");
         let (connected_, locked) = match iota_ledger::get_ledger(
+            0x107a, /* FIXME */
```

Comment:
Does it make any difference for the status which coin type we use here?

Reply:
I'm not sure, I have to check this :thinking:

Reply:
ah yes, I think it makes a difference because when the app is split into iota and shimmer, getting the ledger object with coin_type 0x107a from the shimmer app will fail because the app won't support this :thinking:

Reply:
I'll think about what we can change here

Reply:
If the coin type is required we can add it as parameter to the method

Reply:
changed in: https://github.com/iotaledger/iota.rs/pull/1023/commits/3da725711c6d209e6a9474ce7df435b25623a217